airflow/providers/google/cloud/operators/mlengine.py

```diff
@@ -852,6 +853,23 @@ def execute(self, context):
         )
 
 
+class AIPlatformConsoleLink(BaseOperatorLink):
+    """
+    Helper class for constructing AI Platform Console link.
+    """
+    name = "AI Platform Console"
+
+    def get_link(self, operator, dttm):
+        task_instance = TaskInstance(task=operator, execution_date=dttm)
+        gcp_metadata_dict = task_instance.xcom_pull(task_ids=operator.task_id, key="gcp_metadata")
+        if not gcp_metadata_dict:
+            return ''
+        job_id = gcp_metadata_dict['job_id']
```

Comment:
Will this key be always present?

Reply:
Yes. A dictionary is created and immediately saved. No operations are performed on it, so it always has one structure.